client.c

```diff
@@ -222,7 +220,8 @@ void clientmgr (void *arg)
 
   UNUSED_ARG(arg);
 
-  /* Initialize invalid addresses. Remainder of members is already initialized to 0. */
+  /* Initialize invalid addresses */
+  memset(invalidAddresses, 0, sizeof(invalidAddresses));
```

Comment:
Are you sure about that for every compiler on all platforms that binkd is
created for. And it hardly hurts performance... ;)


On Tue, Jan 19, 2016 at 4:47 PM, Pavel <notifications@github.com> wrote:

> In client.c <https://github.com/pgul/binkd/pull/5#discussion_r50128670>:
>
> > @@ -222,7 +220,8 @@ void clientmgr (void *arg)
> >
> >    UNUSED_ARG(arg);
> >
> > -  /* Initialize invalid addresses. Remainder of members is already initialized to 0. */
> > +  /* Initialize invalid addresses */
> > +  memset(invalidAddresses, 0, sizeof(invalidAddresses));
>
> I think it's not needed because global and static variables are always
> initialized by zero.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pgul/binkd/pull/5/files#r50128670>.
>



-- 
Met vriendelijke groet / Best regards,
Wilfred van Velzen